src/pyfmi/fmi3.pyx

```diff
@@ -855,7 +883,15 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
         return FMI3_Type(int(variable_data_type))
 
     cdef _get_variable_description(self, FMIL3.fmi3_import_variable_t* variable):
-        cdef FMIL3.fmi3_string_t desc = FMIL3.fmi3_import_get_variable_description(variable)
+        cdef FMIL3.fmi3_string_t desc = <FMIL3.fmi3_string_t>FMIL3.fmi3_import_get_variable_description(variable)
+        if desc == NULL:
+            desc = ""
+        desc = <FMIL3.fmi3_string_t>desc
```

Comment:
And related to above, do we need to cast this again if we already did it on line 886?

Reply:
No, fixed.